Sources/Java2SwiftLib/JavaClassTranslator.swift

```diff
@@ -582,10 +612,33 @@ extension JavaClassTranslator {
     )
     let fieldAttribute: AttributeSyntax = javaField.isStatic ? "@JavaStaticField" : "@JavaField";
     let swiftFieldName = javaField.getName().escapedSwiftName
-    return """
+
+    if let optionalType = swiftTypeName.optionalWrappedType() {
+      let setter = if !javaField.isFinal {
+      """
+      
+        set {
+          \(swiftFieldName) = newValue?.toJavaOptional()
+        }
+      """
+      } else { "" }
```

Comment:
The formatting here is somewhat confusing with the else not being indented, what does swift-format suggest here (we don't require it in CI since we have an issue with it hanging on CI... but we need to fix that).